behat/tests/features-v2/client-management/admin/search-client.feature

```diff
@@ -0,0 +1,34 @@
+@v2 @admin-client-search
+Feature: Admin - Client Search
+
+  Scenario: An admin user searches for a client by first name
+    Given an admin user accesses the admin app
+    When I navigate to the admin clients search page
+    And I search for an existing client by their first name
```

Comment:
we need something in here for 'I search for non existing client' to check the correct text is displayed

Reply:
Good catch - will add in.